src/controller/blogsController.ts

```diff
@@ -0,0 +1,66 @@
+import { Request, Response } from "express";
+import {sql} from '@vercel/postgres'
+
+class BlogController {
+    async getAllBlogs(req: Request, res: Response) {
+        try {
+            const blogs = await sql `SELECT * FROM blogs`;
+            res.status(200).json(blogs)
+        } catch (error) {
+              console.error(error)
+              res.status(500).send('Internal Server Error')  
+        }     
+    }
+
+    async getBlogById(req: Request, res: Response) {
+        const {id} = req.params
+
+        try {
+            const blog = await sql `SELECT * FROM blogs WHERE id = ${id}`
+            res.status(200).json(blog)
+        } catch (error) {
+            console.error(error)
+            res.status(404).send('Blog not found')
+        }
+    }
+
+
+    async createBlog(req: Request, res: Response) {
+        const {title, discription, content, author } = req.body;
+
+        try {
+            const newBlog = await sql`INSERT INTO blogs (title, description, content, author) VALUES (${title}, ${discription}, ${content}, ${author}) RETURNING *`;
```

Comment:
inserting into table isn't going to return anything so you don't need a variable